docs/cookbook/presigned-put-upload-via-browser.md

```diff
@@ -0,0 +1,98 @@
+# Upload files from browser using pre-signed URLs [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/minio/minio?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+Using client-side JavaScript and a Node.js application, you can allow users to
+upload files to the database using pre-signed URLs. The following is an
+annotated example of this using [minio-js](https://github.com/minio/minio-js).
+
```

Comment:
you can say:
Using presigned URLs you can allow Web-browser to upload a file directly to S3 without exposing S3 credentials to the browser.